apps/sms/test/marionette/lib/messages_activity_caller.js

```diff
@@ -60,15 +60,16 @@
           client.helper.waitForElement(SELECTORS.pickContactButton).tap();
         },
 
-        sendMessage: function(number, body) {
+        sendMessage: function(parameters) {
           var button = client.helper.waitForElement(
             SELECTORS.sendMessageButton
           );
 
-          button.scriptWith(function(element, number, body) {
-            element.dataset.number = number;
-            element.dataset.body = body;
-          }, [number, body]);
+          button.scriptWith(function(element, parameters) {
+            Object.keys(parameters).forEach(function(parameterKey) {
+              element.dataset[parameterKey] = parameters[parameterKey];
+            });
```

Comment:
you mean in ```executeScript```? If so we should duplicate it in every test that uses this test app...

My idea was to have real test app, so that if test fails for some reason I can tap on buttons manually in b2g while it's run and investigate the problem - it helped me a lot :)

Anyway we can discuss and change it later if you want